butterknife-compiler/src/test/java/butterknife/OnClickTest.java

```diff
@@ -659,8 +660,9 @@
     );
 
     assertAbout(javaSource()).that(source)
+        .withCompilerOptions("-Xlint:-processing")
```

Comment:
This compiler option is only necessary for sources that contain unprocessed annotations (like `ColorInt`, `Nullable`, `Optional`, …). Without [this line](http://stackoverflow.com/a/26533843/1891728) this test would fail with
```
java.lang.AssertionError: Expected 0 warnings, but found the following 1 warnings:
No processor claimed any of these annotations: android.support.annotation.ColorInt
```

If you want I can add this line to all tests to be consistent throughout the codebase.